app/models/profile.rb

```diff
@@ -7,8 +7,6 @@ class Profile < ApplicationRecord
   has_many :requests, dependent: :destroy
   has_many :favourites, dependent: :destroy
 
```

Comment:
This probably needs:

```suggestion
has_many :profile_languages, dependent: :destroy
```

Reply:
This should be on line 3 already

Reply:
My apologies – those lines were collapsed in my view here on Github. All good.